### DISCUSSION TITLE
Image type extension not supporting svg

### DISCUSSION THREAD

**Discussion by dnmahendra** [keystonejs/keystone#8203] [Questions] [answered]
I see this line of code in KeystoneJS context.d.ts file,

`export declare type ImageExtension = 'jpg' | 'png' | 'webp' | 'gif';
`
I am just trying to understand why svg is not supported? Not able to upload svg type images.

**Comment by linus-ha** [comment, marked as answer]
#6715

**Reply by dcousens** [reply]
And now https://github.com/keystonejs/keystone/discussions/8913
